gradle/libs.versions.toml

```diff
@@ -22,10 +22,10 @@ micronaut-validation = "4.11.0"
 
 # Frameworks
 
-managed-vertx = "4.5.21"
+managed-vertx = "5.0.5"
```

Comment:
There is PR to upgrade vertx but is needed for Hibernate Reactive so can do it here.